pkg/rancher-desktop/utils/paths.ts

```diff
@@ -103,7 +103,7 @@ export class WindowsPaths implements Paths {
 }
 
 // Gets the path to rdctl. Returns null if rdctl cannot be found.
-function getRdctlPath(): string | null {
+export function getRdctlPath(): string | null {
```

Comment:
I don't love exporting this, but I guess that will be fine until we fix up `resources.ts`…